docs/12_download.md

```diff
@@ -64,9 +64,20 @@ application, e.g.:
 `dandi download https://api.dandiarchive.org/api/dandisets/000023/versions/0.210914.1900/assets/1a93dc97-327d-4f9c-992d-c2149e7810ae/download/`
 
 
-**Hint:** `dandi download` supports a number of resource identifiers to point to a Dandiset, folder, or file.  Providing
+**Hint:** `dandi download` supports a number of [Resource Identifiers](https://dandi.readthedocs.io/en/latest/ref/urls.html#resource-ids) to point to a Dandiset, folder, or file.  Providing
 an incorrect URL (e.g. `dandi download wrongurl`) will provide a list of supported identifiers.
 
+### Download a specific file and preserve the directory tree of the Dandiset
```

Comment:
it is not that the tree is preserved, but you would also get `dandiset.yaml` which would include the information about dandiset (since folder name does not mean anything).

IMHO should be adjusted to expand on that.

Reply:
Thank you, @yarikoptic.  I have updated the section title.  Please let me know what you think.